cli/tests/fixture/mod.rs

```diff
@@ -35,7 +35,7 @@ pub(crate) fn new_test_with_path(path: &str) -> Test {
     Test {
         subg: test,
         _dir: dir,
-        pool_path: pool_path,
+        pool_path,
```

Comment:
I believe this issue was on `main`, is the github action not enforcing clippy warnings?

Reply:
It should be now. It might only check changed files or something.